contracts/oracles/BoundValidator.sol

```diff
@@ -56,8 +56,6 @@ contract BoundValidator is AccessControlledV8, BoundValidatorInterface {
      * @custom:event Emits ValidateConfigAdded for each validation config that is successfully set
      */
     function setValidateConfigs(ValidateConfig[] memory configs) external virtual {
-        _checkAccessAllowed("setValidateConfigs(ValidateConfig[])");
```

Comment:
We should remove the granted permission also from the VIP: https://github.com/VenusProtocol/vips/pull/4